riak/transports/pbc/transport.py

```diff
@@ -213,7 +217,12 @@ def ts_delete(self, table, key):
 
     def ts_query(self, table, query, interpolations=None):
         req = riak.pb.riak_ts_pb2.TsQueryReq()
-        req.query.base = str_to_bytes(query)
+
+        q = query
+        if '{table}' in q:
```

Comment:
This is a new feature that I don't necessarily have to include. It eliminates the redundancy of specifying the table name as the first parameter to  `ts_query()` as well as within the query string. However, if the query string is itself a format string, the user must remember to escape the parameter via `{{table}}` as you can see in the tests.